scripts/kconfig/kconfigfunctions.py

```
@@ -415,6 +415,27 @@ def dt_node_str_prop_equals(kconf, _, path, prop, val):
 
     return "n"
 
+def dt_node_array_prop_len(kconf, _, label, prop):
+    """
+    This function takes a 'label' and looks for an EDT node for that label. If
+    it finds an EDT node, it will look to see if that node has a property
+    by the name of 'prop'.  If the 'prop' exists and is an array type it will
+    return the length of its array as a string integer value, or 0 if the
+    property does not exist.
```

```suggestion
    return the length of its array as a string integer value. Otherwise, it returns 0.
```

0 is also returned if the node itself does not exist; this makes that a bit clearer.